pkg/collector/corechecks/networkpath/networkpath.go

```diff
@@ -69,18 +69,18 @@ func (c *Check) Run() error {
 	path.Namespace = c.config.Namespace
 
 	// Add tags to path
-	commonTags := append(utils.GetCommonAgentTags(), c.config.Tags...)
 	path.Source.Service = c.config.SourceService
 	path.Destination.Service = c.config.DestinationService
-	path.Tags = commonTags
+	path.Tags = c.config.Tags
```

Comment:
note for reviewers: We don't need tags from `utils.GetCommonAgentTags()` on the path since it already have agent host (source.hostname) and agent version.